packages/web-forms/src/components/form-layout/FormQuestion.vue

```diff
@@ -25,15 +25,33 @@ const isRangeNode = (node: ControlNode) => node.nodeType === 'range';
 const isTriggerNode = (node: ControlNode) => node.nodeType === 'trigger';
 const isUploadNode = (node: ControlNode) => node.nodeType === 'upload';
 
-const submitPressed = inject<boolean>('submitPressed', false);
+const submitPressed = inject<Ref<boolean>>('submitPressed', ref(false));
```

Comment:
Changing from `false` to `ref(false)` to match [the declared type.](https://github.com/getodk/web-forms/blob/49a21590e2fe67be6a3e02065e4f29becc3c0409/packages/web-forms/src/components/OdkWebForm.vue#L151)